### PR TITLE
fsevents: use FlagNoDefer for FSEventStreamCreate

### PR DESCRIPTION
**Pull request by indutny**
Otherwise `FSEventStreamCreate()` will coalesce events, even if they're
happening in the interval, bigger than supplied `latency`. In other
words, if this flag is not set events will happen in separate callback
only if there was a delay bigger than `latency` between two consequntive
events.

### CODE REVIEW

File: src/unix/fsevents.c [hunk 1]
```diff
@@ -316,10 +316,10 @@ static int uv__fsevents_create_stream(uv_loop_t* loop, CFArrayRef paths) {
   ctx.release = NULL;
   ctx.copyDescription = NULL;
 
-  latency = 0.15;
+  latency = 0.05;
 
   /* Set appropriate flags */
```
Comment (bnoordhuis) [inline src/unix/fsevents.c]: Update the comment to explain why you picked these particular flags?

File: test/test-fs-event.c [hunk 2]
```diff
@@ -36,11 +36,21 @@
 #endif
 
 static uv_fs_event_t fs_event;
+static const char* file_prefix = "fsevent-";
```
Comment (bnoordhuis) [inline test/test-fs-event.c]: Make this const char[] and use sizeof below.

**Comment by indutny** [comment]
Is that all I should fix there?

**Comment by bnoordhuis** [comment]
Yes.  Were you expecting more?

**Comment by indutny** [comment]
All fixed, please review grammar! ;)

**Comment by bnoordhuis** [comment]
LGTM.  Apropos the commit log, it's 'consecutive.'

**Comment by indutny** [comment]
Thanks, landed in 0f5c28b.

**Pull request closed without merging**